### ISSUE TITLE
Help me, Please to have hlsdl binaries (for Windows), after each source files update!!!

### ISSUE THREAD

**Issue by kokolino2** [selsta/hlsdl#49]
Please, could someone tell me where I can find regularly updated binaries/windows versions (like source files are updated), of this perfect program? Because I can not make such versions using this description "https://github.com/samsamsam-iptvplayer/hlsdl/blob/master/msvc/BUID_WINDOWS.txt'. I just do not know how to do all this!!! I am sure I can not do this!!!

Please, HELP me!!!!!

**Comment by selsta** [comment]
Sorry, I don’t use Windows :/

**Comment by kokolino2** [comment]
Obviously, it's too hard for someone to make builds for Windows, since there are no any binaries, anyhere! I doubt if it is even possible, at all! Unfortunately, 'e2iplayer', closed his page where he (or she) uploaded some early versions of these binaries!!!

Thank you, for answering, 'selsta'!!!
Could you tell me someone who can make these binaries?

**Comment by d3ibis** [comment]
"it's too hard for someone to make builds for Windows" - How come ?! did u try ? its mentioned in all steps, what to download and how to comply...

Dear @selsta , I tried on Mac b4 compiling a bin for windows just to make sure its working, faced some issues I'll open ticket for it.

**Comment by selsta** [comment]
@d3ibis macOS build should be working again, see https://github.com/selsta/hlsdl/runs/330160910

**Comment by briggeml** [comment]
Today's build...
[hlsdl_v0.26-x64-static.zip](https://github.com/selsta/hlsdl/files/4338556/hlsdl_v0.26-x64-static.zip)

**Comment by kokolino2** [comment]
Hello! And THANK You, 'briggeml'!!!

Tell me please, why when I start your 'hlsld.exe', it opens this error message: "Access is denied"?
I am on WIndows XP, 64bit operating system!
Are also, any dependencies?

Thank You, again, for your work and efforts in creating Windows binaries!!!

**Comment by briggeml** [comment]
> Tell me please, why when I start your 'hlsld.exe', it opens this error message: "Access is denied"?

Probably insufficient rights for file/folder

> I am on WIndows XP, 64bit operating system!

I haven't tested the build on this OS, only on Win7x64

> Are also, any dependencies?

No.

**Comment by kokolino2** [comment]
Thank You, Very MUCH, 'briggeml'! I will try it!!!

**Comment by mblackshore** [comment]
@briggelm Does your version sort out the sample-128 problem?
https://github.com/selsta/hlsdl/issues/31#issuecomment-435191950

I ran yours and it outputs a .ts file as expected..but it does not play.

**Comment by briggeml** [comment]
> @briggelm Does your version sort out the sample-128 problem?
> [#31 (comment)](https://github.com/selsta/hlsdl/issues/31#issuecomment-435191950)

My build as well as the sources in this branch do not use ffmpeg at all. The sample-128 problem as far as I see was solved long ago in these sources.

> I ran yours and it outputs a .ts file as expected..but it does not play.

I guess the problem is something else... I'll attach the last build.

[hlsdl_v0.27-x64-static.zip](https://github.com/selsta/hlsdl/files/4673224/hlsdl_v0.27-x64-static.zip)

**Comment by Voskrese** [comment]
[hlsdl_v0.27-x64-dynamic-release.zip.zip](https://github.com/selsta/hlsdl/files/4787198/hlsdl_v0.27-x64-dynamic-release.zip.zip)

**Comment by kokolino2** [comment]
'Voskrese', THANK You, VERY MUCH!!!

**Comment by jonirrings** [comment]
Try to use Github-Action to build binaries?
